binaries/package_data.yaml

```diff
@@ -0,0 +1,14 @@
+macosx_10_9_x86_64:
+- packager-osx
+- ffprobe-osx-x64
+- ffmpeg-osx-x64
+manylinux1_x86_64:
```

Comment:
This file looks incomplete (no packager binary names for linux) and has no license header.  Is it generated?  Should this be in .gitignore?